src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspaceManager.cs

```diff
@@ -152,7 +152,9 @@ public Assembly LoadAssembly(string assemblyFullName, string? codeBasePath)
                 var assemblyName = new AssemblyName(assemblyFullName);
                 if (!string.IsNullOrEmpty(codeBasePath))
                 {
+#pragma warning disable SYSLIB0044 // Type or member is obsolete
```

Comment:
@tmat CodeBase is deprecated in .NET 8, any idea what's the right fix here would be?

Reply:
Assembly.LoadFromFile?

Reply:
Or actually, maybe just ignore this? IDK what MEF uses this for.

Reply:
hmm, yea, I don't see this method got called. I guess suppressing it is enough